Runtime/Scripts/RTCPeerConnection.cs

```diff
@@ -330,6 +341,18 @@ static void OnFailureSetSessionDesc(IntPtr ptr)
                 connection.m_opSetDesc.Done();
             }, null);
         }
+
+        [AOT.MonoPInvokeCallback(typeof(DelegateSetSDFailure))]
```

Comment:
This should be used value `DelegateCollectStats` not `DelegateSetSDFailure`

Reply:
Ok